jarviz-lib/src/main/java/com/vrbo/jarviz/AnalyzeCommand.java

```diff
@@ -164,7 +165,9 @@ private static void reportError(final Exception e) {
             System.err.println("Analyze failed");
             e.printStackTrace();
         } else {
-            System.err.println("Analyze failed: " + e.getMessage());
```

Comment:
Will bring this in

Reply:
I merge your new master into this branch to resolve merge conflicts.